### DISCUSSION TITLE
what is the ports needed by Elastic Agent Installer?

### DISCUSSION THREAD

**Discussion by toxinliquid** [Security-Onion-Solutions/securityonion#11948] [2.4] [answered]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

20

### Storage for /

250

### Storage for /nsm

50

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

what is the ports needed by Elastic Agent Installer?
I need to to create a quick script and push it with ad to unlock those ports before pushing the installer 
Thank you

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by robbiemarshall** [comment, marked as answer]
You can find all necessary ports for Security Onion in the documentation found here:  https://docs.securityonion.net/en/2.4/firewall.html

**Reply by toxinliquid** [reply]
Thank you ,exactly what i needed
